### DISCUSSION TITLE
page Frontmatter

### DISCUSSION THREAD

**Discussion by ShaoClean** [vuepress-theme-hope/vuepress-theme-hope#3097] [Q&A] [answered]
问题：
每个md文件开头都需要写一下Frontmatter配置，于是想做一个vue组件（MyHeader.vue），如图：
<img width="358" alt="image" src="https://user-images.githubusercontent.com/98806795/235624709-8f5938e7-9042-4875-87ff-436f3e0247f7.png">
引入到需要的md文件时，如图：
<img width="225" alt="image" src="https://user-images.githubusercontent.com/98806795/235624928-1e03883b-7527-467a-b16f-b792bcddf26a.png">
结果如图：
<img width="394" alt="image" src="https://user-images.githubusercontent.com/98806795/235624157-de5e1103-d9d5-44d7-8520-c86b1f342422.png">

预期：
<img width="656" alt="image" src="https://user-images.githubusercontent.com/98806795/235625455-836ea28e-34b8-48c0-814e-8930433bc92e.png">

**Comment by Mister-Hope** [comment, marked as answer]
You need to understand what's frontmatter first.

You can change it programly in `extendsPage` lifecycle

**Reply by ShaoClean** [reply]
ok i will try
